extensions/functions_datetime.yaml

```diff
@@ -57,8 +73,7 @@ scalar_functions:
           - value: date
           - value: interval_day
         return: timestamp
-  -
-    name: "add_intervals"
+  - name: "add_intervals"
```

Comment:
Why did you reformat all of these? Let's not do reformatting changes like this. To be consistent, if we think they are important let's add to a linter check as part of precommit.